docs/getting-started-guides/docker-multinode.md

```diff
@@ -0,0 +1,261 @@
+
+```sh
+sudo docker run --net=host -d -v /var/run/docker.sock:/var/run/docker.sock  gcr.io/google_containers/hyperkube:v0.14.1 /hyperkube kubelet --api_servers=http://localhost:8080 --v=2 --address=0.0.0.0 --enable_server --hostname_override=127.0.0.1 --config=/etc/kubernetes/manifests-multi
+```
+
+### Also run the service proxy
+```sh
+sudo docker run -d --net=host --privileged gcr.io/google_containers/hyperkube:v0.14.1 /hyperkube proxy --master=http://127.0.0.1:8080 --v=2
+```
+
+## Adding a new node
+These instructions are very similar to the master set-up above, but they are duplicated for clarity.
+You need to repeat these instructions for each node you want to join the cluster.
+We will assume that the IP address of this node is ```${NODE_IP}``` and you have the IP address of the master in ```${MASTER_IP}```
+
+
+### Set up Flannel on the worker node
+As before, the Flannel daemon is going to provide network connectivity.
+
+#### Set up a boostrap docker:
+As above, we need a second instance of the Docker daemon running to bootstrap the flannel networking.
+
+Run:
+```sh
+sudo sh -c 'docker -d -H unix:///var/run/docker-bootstrap.sock -p /var/run/docker-bootstrap.pid --iptables=false --ip-masq=false --bridge=none --graph=/var/lib/docker-bootstrap 2> /var/log/docker-bootstrap.log 1> /dev/null &'
+```
+
+#### Run flannel
+
+Now run flanneld itself, this call is slightly different from the above, since we point it at the etcd instance on the master.
+```sh
+sudo docker -H unix:///var/run/docker-bootstrap.sock run -d --net=host --privileged -v /dev/net:/dev/net quay.io/coreos/flannel:0.3.0 /opt/bin/flanneld --etcd-endpoints=http://${MASTER_IP}:4001
+```
+
+#### Edit the docker configuration
+You now need to edit the docker configuration to activate new flags.  Again, this is system specific.
+
+This may be in ```/etc/docker/default``` or ```/etc/systemd/service/docker.service``` or it may be elsewhere.
+
+Regardless, you need to add the following to the docker comamnd line:
+```sh
+--bip=${FLANNEL_SUBNET} --mtu=${FLANNEL_MTU}
+```
+
+#### Remove the existing Docker bridge
+Docker creates a bridge named ```docker0``` by default.  You need to remove this:
+
+```sh
+sudo /sbin/ifconfig docker0 down
+sudo brctl delbr docker0
+```
+
+You may need to install the ```bridge-utils``` package for the ```brctl``` binary.
+
+#### Restart Docker
+Again this is system dependent, it may be:
+
+```sh
+sudo /etc/init.d/docker start
+```
+
+it may be:
+```sh
+systemctl start docker
+```
+
+### Start Kubernetes on the worker node
+#### Run the kubelet
+Again this is similar to the above, but the ```--api_servers``` now points to the master we set up in the beginning.
+
+```sh
+sudo docker run --net=host -d -v /var/run/docker.sock:/var/run/docker.sock  gcr.io/google_containers/hyperkube:v0.14.1 /hyperkube kubelet --api_servers=http://${MASTER_IP}:8080 --v=2 --address=0.0.0.0 --enable_server --hostname_override=$(hostname -i)
+```
```

Comment:
If we plan on using hostname here the user must ensure DNS is setup correctly. This won't be the case in a lot of environments.